### ISSUE TITLE
Original / new asset independent paths

### ISSUE THREAD

**Issue by Katharsas** [Katharsas/GMM#80]
Requirement for #78 

The subpath to an original asset should not be forced to be the same as the subpath in SVN for a new asset.
The assumption that it should simply be the same path was valid when GMM was the only user to save new assets, but because of #78 , users may choose their own location for new assts by changing the SVN repo.

This also means that storing preview files in the new asset folders makes no sense because they also include previews for original files, and the new asset folder may not yet exist while original preview is already needed. Splitting previews into original/new does not work, because original folder structure has no place for previews.
So basically there needs to be a preview folder which holds one previewfolder per original/new asset (totally flat folder structure). Basically a map: assetName -> previews

Original previews will be only generated, when no original previews exist. Original assets are expected to not change, and original previews are thus expected to get deleted manually if at all. New previews will be created/updated/deleted based on SVN changes and are expected to always correspond to SVN files.

**Comment by Katharsas** [comment]
TODO:
If an asset task exists (e.g. because an original asset exists), but a new asset folder does not yet exists, the user must be asked where to create the asset folder in the vcs when user tries to upload first file (can be asset or non asset).

**Comment by Katharsas** [comment]
TODO:
Add CSS for folder creation dialog and and add vertical scroll for big file tree view.
